test/abilities/magic_guard.test.ts

```diff
@@ -195,8 +195,8 @@ describe("Abilities - Magic Guard", () => {
      * - The player Pokemon (with Magic Guard) has not taken damage from poison
      * - The enemy Pokemon (without Magic Guard) has taken damage from poison
      */
-    expect(leadPokemon.status!.effect).toBe(StatusEffect.POISON);
-    expect(enemyPokemon.status!.effect).toBe(StatusEffect.POISON);
+    expect(leadPokemon.getStatusEffect()).toBe(StatusEffect.POISON);
+    expect(enemyPokemon.getStatusEffect()).toBe(StatusEffect.POISON);
```

Comment:
```suggestion
    expect(leadPokemon.getStatusEffect(true)).toBe(StatusEffect.POISON);
    expect(enemyPokemon.getStatusEffect(true)).toBe(StatusEffect.POISON);
```
is it intended that these are not ignoring comatose but other tests are?
I guess it doesn't change anything in this case, just wondering about consistency between tests